developers/weaviate/client-libraries/python.md

```diff
@@ -308,6 +318,19 @@ We are evaluating modifing this to send multiple requests by matches in the futu
 </TabItem>
 </Tabs>
 
+#### `insert_many` with DataObjects
+
+The `insert_many` function allows you to pass in a list of `DataObject` instances or a list of dictionaries. This is useful if you want to specify additional information to the properties, such as cross-references, object uuid, or a custom vector.
```

Comment:
Consider: 
The `insert_many` function takes a list

Reply:
Updated